app/cdap/components/Snackbar/Components/Transition/styles.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { makeStyles } from '@material-ui/core';
+
+export const useStyles = makeStyles((theme) => ({
+  warningIcon: {
+    color: '#ffffff',
+    marginRight: 10,
+    fontSize: 'x-large',
+  },
+  successIcon: {
+    fontSize: 'xx-large',
+    color: '#ffffff',
+    paddingRight: 14,
+    position: 'relative',
+    bottom: 4,
+  },
+  successLabel: {
+    color: '#ffffff',
+    fontSize: '16px !important',
```

Comment:
use number instead of number with pixel

Reply:
since we have used important, it is not required to convert into number.